admin/class-surveyfunnel-lite-admin.php

```diff
@@ -576,6 +576,7 @@ public function surveyfunnel_lite_get_configuration_data() {
 		$post_meta = get_post_meta( $post_id, 'surveyfunnel-lite-data', true );
 		$data      = array(
 			'configure' => $post_meta['configure'],
+            'proActive' => apply_filters( 'surveyfunnel_pro_activated', false ),
```

Comment:
use variable naming conventions as per WordPress standards: For eg: proActive => pro_active